### PR TITLE
Simplify pull request template for users

### PR DESCRIPTION
**Pull request by sharwell**
It looks like this now:

<details><summary>Ask Mode template not completed</summary>

### Customer scenario

What does the customer do to get into this situation, and why do we think this
is common enough to address for this release.  (Granted, sometimes this will be
obvious "Open project, VS crashes" but in general, I need to understand how
common a scenario is)

### Bugs this fixes

(either VSO or GitHub links)

### Workarounds, if any

Also, why we think they are insufficient for RC vs. RC2, RC3, or RTW

### Risk

This is generally a measure our how central the affected code is to adjacent
scenarios and thus how likely your fix is to destabilize a broader area of code

### Performance impact

(with a brief justification for that assessment (e.g. "Low perf impact because no extra allocations/no complexity changes" vs. "Low")

### Root cause analysis

How did we miss it?  What tests are we adding to guard against it in the future?

### How was the bug found?

(E.g. customer reported it vs. ad hoc testing)

### Test documentation updated?

If this is a new non-compiler feature or a significant improvement to an existing feature, update https://github.com/dotnet/roslyn/wiki/Manual-Testing once you know which release it is targeting.

</summary>

### CODE REVIEW

**Comment by sharwell** [comment]
For reference, this is the one we have to fill out internally using data from the template:

> **Customer and scenario info**
> 
> * Who is impacted by this bug: 
> * What is the customer scenario and impact of the bug: 
> * What is the workaround: 
> * How was the bug found:​ 
> * If this fix is for a regression - what had regressed, when was the regression introduced, and why was the regression originally missed: 
> * ​What is a customer rea​dy description of change should we need to document this: 
> 
> **Bug fix details**
> 
> * Overview of the fix: 
> * Which VS products, SKUs, or components need the fix: 
> * What branch does the fix go into: 
> * What feature areas are impacted by the fix: 
> * Is this checkin an insertion? If so, provide the link to the val build:​ 
> * What is the risk of the code change: 
> * Does this change have impact on anything else (e.g., setup, perf, stress, ux, visual freeze, go live, breaking change, samples): 
> * How is this fix rolling forward to the next major release / sprintly train (e.g., Dev16):​ 
> 
> **​Validation**
> 
> * What testing/validation was done on the fix, and what were the results: 
> * (Binary/Component Insertions only) Have you validated that your insertion does not regress DDRIT / RPS: 
> * (Singleton changes only) What validation has been done to ensure SxS functionality with prior releases (see guidance): 
> * Which engineering partner teams have signed off on the fix: 
> * Who code reviewed the fix:

**Comment by sharwell** [comment]
> Why not update the text to what we have to use internally to enable copy/paste?

Several items in the text would never make sense for non-Microsoft employees. It ended up being so many that it didn't make sense to change from what we already had.

* What is a customer rea​dy description of change should we need to document this
* Which VS products, SKUs, or components need the fix
* What branch does the fix go into
* What feature areas are impacted by the fix
* Is this checkin an insertion? If so, provide the link to the val build
* Does this change have impact on anything else (e.g., setup, perf, stress, ux, visual freeze, go live, breaking change, samples)
* How is this fix rolling forward to the next major release / sprintly train (e.g., Dev16)
* (Binary/Component Insertions only) Have you validated that your insertion does not regress DDRIT / RPS
* (Singleton changes only) What validation has been done to ensure SxS functionality with prior releases (see guidance)
* Which engineering partner teams have signed off on the fix
* Who code reviewed the fix

**Comment by sharwell** [comment]
@dotnet-bot retest windows_debug_vs-integration_prtest please

**Comment by sharwell** [comment]
@dotnet-bot retest windows_release_vs-integration_prtest please

**Comment by sharwell** [comment]
@jasonmalinowski can I get a review here

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,37 +1,44 @@
...
+### How was the bug found?
 
 (E.g. customer reported it vs. ad hoc testing)
 
-**Test documentation updated?**
+### Test documentation updated?
 
 If this is a new non-compiler feature or a significant improvement to an existing feature, update https://github.com/dotnet/roslyn/wiki/Manual-Testing once you know which release it is targeting.
+
+</summary>
```
Comment (jasonmalinowski) [inline .github/PULL_REQUEST_TEMPLATE.md]: Is this supposed to be `</details>`? Otherwise it looks like you have two close summary tags.

**Pull request merged**